### DISCUSSION TITLE
How to use train_text_to_image_sdxl.py with unlabed data

### DISCUSSION THREAD

**Discussion by khayamgondal** [huggingface/diffusers#7427] [Q&A]
How can I use `examples/text_to_image/train_text_to_image_sdxl.py` with images without captions?
 Dreambooth has a version for sdxl_lora but I am looking for a training script that uses sdxl without lora.

**Comment by sayakpaul** [comment]
Generate the captions using multi-modal models. 

> Dreambooth has a version for sdxl_lora but I am looking for a training script that uses sdxl without lora.

Feel free to repupose this script: https://github.com/huggingface/diffusers/blob/main/examples/dreambooth/train_dreambooth.py.
